### PR TITLE
fix badge link

### PR DESCRIPTION
**Pull request by Zt-freak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 See [License](LICENSE)
 
-[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](code_of_conduct.md)
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](CONTRIBUTING.md)
```
Comment (Colliflower) [inline README.md:7]: Shouldn't this be [CODE_OF_CONDUCT.md](https://github.com/hsutter/cppfront/blob/main/CODE_OF_CONDUCT.md)?

Comment (Zt-freak) [reply README.md:7]: oh true, the capitalisation broke the link and I coped the wrong markdown file.

Comment (Zt-freak) [reply README.md]: it should have the right file name now

Comment (Zt-freak) [reply README.md]: you know what, I'll make a different branch and do it correctly in one go. no need for two commits.

**Pull request closed without merging**